argocd/resource_argocd_application_test.go

```diff
@@ -1222,41 +1222,53 @@ resource "argocd_application" "helm_file_parameters" {
 func testAccArgoCDApplicationKustomize(name string) string {
```

Comment:
This test is failing for `v2.8.x` as the feature was only added in `v2.9.0`. Can you please add a feature flag for this functionality and split the newer config out into a separate test? See [ProjectSourceNamespaces](https://github.com/search?q=repo%3Aoboukili%2Fterraform-provider-argocd%20ProjectSourceNamespaces&type=code) for reference.

Reply:
Hi @onematchfox , thanks for reviewing the PR. I've updated the pr with the commented changes